functions/Get-DbaDbPartitionScheme.ps1

```diff
@@ -66,6 +74,8 @@ function Get-DbaDbPartitionScheme {
         [PSCredential]$SqlCredential,
         [object[]]$Database,
         [object[]]$ExcludeDatabase,
+        [Alias("Name")]
+        [string[]]$PartitionScheme,
```

Comment:
partition scheme is added to Get but only Remove is tested

Reply:
That was my attempt to reset all the things :) I think it's sorted now. Sorry for the delay.

Reply:
no problem for the delay, thanks for the enhancement🫂